common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -693,56 +699,57 @@ def get_problem_html(self, encapsulate=True, demand_hint_text=None, hint_index=0
 
         return html
 
-    def _get_answer_notification(self):
+    def _get_answer_notification(self, render_notifications=False):
```

Comment:
Also, update the documentation of this method to state what the impact of render_notifications is.